### ISSUE TITLE
NavigationItem Controls Out of View on Form Sheet

### ISSUE THREAD

**Issue by zgosalvez** [CosmicMind/Material#386]
On the iPad, if the view controller is presented modally as a form sheet, the left and right controls of navigationItem are out of view. However, for some reason, I can't find it in the view debugger.

**Comment by danieldahan** [comment]
Nice catch. I will take a look at this in the morning. Thank you!

**Comment by danieldahan** [comment]
Can you try the latest and tell me if this persists [1.41.2](https://github.com/CosmicMind/Material/releases/tag/1.41.2).

**Comment by zgosalvez** [comment]
This issue still persists in [1.41.2](https://github.com/CosmicMind/Material/releases/tag/1.41.2).

**Comment by danieldahan** [comment]
Can you send me a sample project with this issue? to support@cosmicmind.io

**Comment by zgosalvez** [comment]
Sent.

**Comment by mohpor** [comment]
@zgosalvez , @danieldahan Can you loop me in? I'm not sure about the setup resulting to this issue. I could reproduce it in my own way, but I'm not sure if that addresses the same problem here.

**Comment by mohpor** [comment]
@zgosalvez Can you test your project against this pod to see if your issue is solved?

```
pod 'Material', :git => 'https://github.com/mohpor/Material.git', :branch => 'development'
```

thanks

P.S.: your sample project works with this patch.

**Comment by zgosalvez** [comment]
@mohpor Your fix works for my actual project. 😀 Awesome! Excited for the official patch release. Thanks, man.

**Comment by mohpor** [comment]
Glad it helped.
There will be an appropriate fix for it ASAP.

**Comment by danieldahan** [comment]
Thank you @mohpor :) And the fix will be out latest tomorrow morning.

**Comment by danieldahan** [comment]
This is fixed in the latest release [1.41.3](https://github.com/CosmicMind/Material/releases/tag/1.41.3)
